packages/@lwc/signals/README.md

```diff
@@ -0,0 +1,75 @@
+# @lwc/signals
+
+This is an experimental package containing the interface expected for signals.
+
+A key point to note is that when a signal is both bound to an LWC class member variable and used on a template,
+the LWC engine will attempt to subscribe a callback to rerender the template.
+
+## Reactivity with Signals
+
+A Signal is an object that holds a value and allows components to react to changes to that value.
+It exposes a `.value` property for accessing the current value, and `.subscribe` methods for responding to changes.
+
+```js
+import { signal } from 'some/signals';
+
+export default class ExampleComponent extends LightningElement {
+    count = signal(0);
+
+    increment() {
+        this.count.value++;
+    }
+}
+```
+
+In the template, we can bind directly to the `.value` property:
+
+```html
+<template>
+    <button onclick="{increment}">Increment</button>
+    <p>{count.value}</p>
+</template>
+```
+
+## Supported APIs
```

Comment:
How quickly is the code in the README gonna get out of date? 😅

Reply:
This problem isn't unique to `@lwc/signals`. I think this is fine for now. Long term, I'd love for these sections of the README to be generated from comments + types of the top-level exposed functions/classes/etc.